conf/machine/imx8ulpevk.conf

```diff
@@ -0,0 +1,20 @@
+#@TYPE: Machine
+#@NAME: i.MX 8ULP EVK
+#@SOC: i.MX8ULP
+#@DESCRIPTION: Machine configuration for NXP i.MX 8ULP Evaluation Kit
```

Comment:
Don't include this file either, we still haven't fully adapted since the machine renames.

Reply:
Thanks @thochstein , let me rework the patchset.

Reply:
@thochstein done!